README.md

```diff
@@ -0,0 +1,94 @@
+- Adding/Removing a repository to Paketo
+- Changes to the contents of the Data Format file listed [here](https://github.com/buildpacks/spec/blob/master/buildpack.md#data-format)
+- Changes that effect contents of the output image
+- Process changes
+- Governance changes
+
+For clarification about where a change fits into this model, please review previous RFCs, or reach
+out on the official Paketo slack [here](packetobuildpacks.slack.com).
+
+### Process
+#### RFCs
+To get an RFC implemented, first the RFC needs to be merged into the `rfcs`[/rfcs] repo. Once an RFC is merged, it's considered 'accepted' and may be implemented in the project. These steps will get an RFC to be considered:
+
+- Fork the RFC repo: <https://github.com/paketo-buildpacks/rfcs>
+- Copy '0000-template.md' to 'accepted/0000-my-feature.md' (where 'my-feature' is descriptive. Don't assign an RFC number yet).
+- Fill in RFC. Any section can be marked as "N/A" if not applicable.
+- Submit a pull request. The pull request is the time to get review of the proposal from the larger community.
+- Build consensus and integrate feedback. RFCs that have broad support are much more likely to make progress than those that don't receive any comments.
+
+Once a pull request is opened, the RFC is now in development and the following will happen:
+
+- It will be labeled as general or specific to a set of teams.
+- Voting members are defined as follows:
+	- If the RFC effects a single team, voting members are Steering Committee Members + Maintainers who are part of the effected team
+	- If the RFC effects multiple teams, voting members are Steering Committee Members + Maintainers who are part of the effected teams. 
+	- IF the RFC effects all teams, voting members are Steering Committee Members + All Maintainers
+
+	The effect of an RFC referenced above can be defined as follows:
+
+	- Changes inside a repo a team owns effect that team
+	- If changes are proposed against a tool or utility, teams that use that utility are effected.
+	- Changes to any specification that apply to a repo effect all teams that use or maintain that repo. 
+
+- The community will discuss as much as possible in the RFC pull request directly. All discussion should be posted on the PR thread.
+- When an RFC is deemed "ready"
+- A Voting member may propose a "motion for final comment period (FCP)" along with a disposition of the outcome (merge, close, or postpone). Before entering FCP, super majority of the voting members must sign off.
+- This step is taken when enough discussion of the trade-offs have taken place and the team(s) is in a position to make a decision.
+- The FCP will last 7 days. If there's unanimous agreement among the team(s), then the FCP can close early.
+- Acceptance requires a super majority of binding votes by voting members in favor. The voting options are the following: Affirmative, Negative, and Abstinence. Non-binding votes are of course welcome. Super majority means 2/3 or greater.
```

Comment:
```suggestion
- Acceptance requires a super majority of binding votes by voting members in favor. The voting options are the following: Affirmative, Negative, and Abstention. Non-binding votes are of course welcome. Super majority means 2/3 or greater.
```

Reply:
Additionally, I don't think I've seen any definition in here of "binding" vs. "non-binding" votes.

Reply:
Non-binding just means you can vote, but don't expect it to count. Only voting members are given votes.